src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -48,8 +47,43 @@ interface VisCardProps {
   onVisTypeSelected: (visType: BaseVisType | VisTypeAlias) => void;
   visType: BaseVisType | VisTypeAlias;
   showExperimental?: boolean | undefined;
+  shouldStretch?: boolean;
 }
 
+const tabs: Array<{ id: 'recommended' | 'legacy'; label: ReactNode; ['data-test-subj']: string }> =
+  [
+    {
+      id: 'recommended',
+      label: i18n.translate('visualizations.newVisWizard.recommendedTab', {
+        defaultMessage: 'Recommended',
+      }),
+      ['data-test-subj']: 'groupModalRecommendedTab',
```

Comment:
nit:
```suggestion
      dataTestSubj: 'groupModalRecommendedTab',
```
no need to have the same case name if you are not going to spread that object into a react component

Reply:
https://github.com/elastic/kibana/pull/195966/commits/17197c21d02b8bbc8e86c8e5135ed8258d4fb177